R/jfa-methods.R

```diff
@@ -804,6 +804,24 @@ plot.jfaEvaluation <- function(x, type = c("estimates", "posterior"), ...) {
       stop(paste0('plot(..., type = "posterior") not supported for method = "', x[["method"]], '" without "prior = TRUE"'))
     }
     p <- plot.jfaPlanning(x, ...)
+  } else if (type == "sequential") {
+    if (is.null(x[["data"]])) {
+      stop(paste0('plot(..., type = "sequential") not supported when data is not supplied'))
+    } else if (is.null(x[["prior"]])) {
```

Comment:
These two statements are not mutually exclusive, so please state them separately without the `else`